cmd/ulxly/ulxly.go

```diff
@@ -822,106 +1086,377 @@ func getDeposits(bridgeServiceDepositsEndpoint string) (globalIndex *big.Int, or
-
-	ulxlyInputArgs.BridgeAddress = depositGetCmd.Flags().String("bridge-address", "", "The address of the lxly bridge")
-	ulxlyInputArgs.InputFileName = ProofCmd.PersistentFlags().String("file-name", "", "The filename with ndjson data of deposits")
-	ulxlyInputArgs.DepositNum = ProofCmd.PersistentFlags().Uint32("deposit-number", 0, "The deposit that we would like to prove")
+	bridgeAssetCommand = &cobra.Command{
+		Use:     "asset",
+		Short:   "send a single deposit of value or an ERC20 into the bridge",
+		Long:    bridgeAssetUsage,
+		PreRunE: prepInputs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return bridgeAsset(cmd)
+		},
+	}
+	bridgeMessageCommand = &cobra.Command{
+		Use:     "message",
+		Short:   "send some value along with call data into the bridge",
+		Long:    bridgeMessageUsage,
+		PreRunE: prepInputs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return bridgeMessage(cmd)
+		},
+	}
+	bridgeMessageWETHCommand = &cobra.Command{
+		Use:     "weth",
+		Short:   "send some WETH into the bridge",
+		Long:    bridgeWETHMessageUsage,
+		PreRunE: prepInputs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return bridgeWETHMessage(cmd)
+		},
+	}
+	claimAssetCommand = &cobra.Command{
+		Use:     "asset",
+		Short:   "perform a claim of a given deposit in the bridge",
+		Long:    claimAssetUsage,
+		PreRunE: prepInputs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return claimAsset(cmd)
+		},
+	}
+	claimMessageCommand = &cobra.Command{
+		Use:     "message",
+		Short:   "perform a claim of a given message in the bridge",
+		Long:    claimMessageUsage,
+		PreRunE: prepInputs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return claimMessage(cmd)
+		},
+	}
+	claimEverythingCommand = &cobra.Command{
+		Use:     "claim-everything",
+		Short:   "attempt to claim any unclaimed deposits",
+		PreRunE: prepInputs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return claimEverything(cmd)
+		},
+	}
+	emptyProofCommand = &cobra.Command{
+		Use:   "empty-proof",
+		Short: "create an empty proof",
+		Long:  "Use this command to print an empty proof response that's filled with zero-valued siblings like 0x0000000000000000000000000000000000000000000000000000000000000000. This can be useful when you need to submit a dummy proof.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return emptyProof()
+		},
+	}
+	zeroProofCommand = &cobra.Command{
+		Use:   "zero-proof",
+		Short: "create a proof that's filled with zeros",
+		Long: `Use this command to print a proof response that's filled with the zero
+		hashes. This values are very helpful for debugging because it would
+		tell you how populated the tree is and roughly which leaves and
+		siblings are empty. It's also helpful for sanity checking a proof
+		response to understand if the hashed value is part of the zero hashes
+		or if it's actually an intermediate hash.`,
```

Comment:
```suggestion
		Long: `Use this command to print a proof response that's filled with the zero
hashes. This values are very helpful for debugging because it would
tell you how populated the tree is and roughly which leaves and
siblings are empty. It's also helpful for sanity checking a proof
response to understand if the hashed value is part of the zero hashes
or if it's actually an intermediate hash.`,
```